### ISSUE TITLE
Chapters do not appear available

### ISSUE THREAD

**Issue by akshimartins** [tachiyomiorg/tachiyomi-extensions#18350]
### Source information

MangaFire 1.4.3

### Source language

Portuguese - BR

### Steps to reproduce

![Screenshot_20231003-091207_Tachiyomi](https://github.com/tachiyomiorg/tachiyomi-extensions/assets/146854709/851c31c8-22f6-4176-9f33-a8ecf42661e3)

### Expected behavior

Show available chapters, but doesn't show any in Portuguese.

### Actual behavior

Chapter count doesn't match. Does not show any chapter in Portuguese, only the English extension.

### Tachiyomi version

0.14.6

### Android version

Android 12

### Other details

The chapters appeared normally in English, all other languages ​​in the extension have the same problem, saying that the chapter count does not match. However, the chapters appeared normal in the web view

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by SkyNalix** [comment]
Fixed if you replace in the first line in  `updateChapterList` in `MangaFire.kt`
```kotlin
        val document = client.newCall(mangaDetailsRequest(manga)).execute().asJsoup()
```
by 
```kotlin
        val request = chapterListRequest(manga.url, chapterType)
        val response = client.newCall(request).execute()
        val result = json.decodeFromString<ResponseDto<String>>(response.body.string()).result
        val document = Jsoup.parse(result)
```
But as of now everytime when you refresh chapter list it make 3 requests, and with this change it will use `chapterListRequest` for a second time, so if anyone know how to manage to use it only once per refresh, please do

**Comment by kaizokuroo** [comment]
> Fixed if you replace in the first line in `updateChapterList` in `MangaFire.kt`

Hello, I'm joining the discussion, I have exactly the same problem on the same source MangaFire but for Japanese (and also tried every languages, only English works).

I wanted to try this solution but I don't know where I can find the file you mentioned and how to edit it. I'm on an unrooted Samsung Galaxy A52. What is the path of the mentioned file?
Thanks in advance

**Comment by alessandrojean** [comment]
This file needs to be edited in a PC and tested by compiling the extension in Android Studio, read the contributing guide. You won't be able to edit in a phone.

**Comment by Opi-Txm** [comment]
Would like to work on this issue :D

**Comment by JoaoPNunes1** [comment]
> This file needs to be edited in a PC and tested by compiling the extension in Android Studio, read the contributing guide. You won't be able to edit in a phone.

Olá, consegui alterar o arquivo MangaFire.kt, mas como faço para compilar em um apk utilizavel?

**Comment by alessandrojean** [comment]
The CONTRIBUTING.md guide has all info you need as mentioned before.
